gradle-tests/olTest/build.gradle

```diff
@@ -1,3 +1,3 @@
 dependencies {
-    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '22.0.0.12'
+    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '23.0.0.3'
```

Comment:
Does there need to be an update to `gradle-tests/wlpTest/build.gradle` for the version?

Reply:
I thought about that.  Could update to 23.0.0.2 which is the latest one that it could be updated to, but I figured leaving it to the last 3, 6, 9, 12 was probably better, but I could go either way.

Reply:
Could also leave them both at 22.0.0.12.  There isn't a `need` to update them.

Reply:
Build failed with 22.0.0.12.  Changing it to 23.0.02 resolved the problem.  So updated it as you were thinking might be needed.